MAUI/Maps/tile-layer.md

```diff
@@ -0,0 +1,216 @@
+
+Defaults to `MapLatLng(0.0, 0.0)`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate="https://tile.openstreetmap.org/{z}/{x}/{y}.png">
+            <map:MapTileLayer.Center>
+                <map:MapLatLng Latitude="27.175014"
+                               Longitude="78.042152">
+                </map:MapLatLng>
+            </map:MapTileLayer.Center>
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+tileLayer.Center = new MapLatLng(27.175014, 78.042152);
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![OSM map center](images/tile-layer/osm_center.png)
+
+## Cache a tile images in application memory
+
+The `CanCacheTiles` property used to decide whether the tile images should be cached in application memory or not. The default value of the `CanCacheTiles` is `false`.
+
+While enabling the `CanCacheTiles` as `true` then need to set the tile server name to maintain the folder to store cache tiles in `MapTileLayer.UrlTemplate` property. The default tile server name to store tile cache is OSM. If you want to change server name you can use the below URL format.
+
+https://example_provider/{z}/{x}/{y}.png?name=serverName
+
+Here, you can replace the serverName as per your wish.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate = "http://api.tomtom.com/map/1/tile/basic/main/{z}/{x}/{y}.png?key=subscription_key?name=tomtom"
+                          CanCacheTiles= "True">
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "http://api.tomtom.com/map/1/tile/basic/main/{z}/{x}/{y}.png?key=subscription_key?name=tomtom";
+tileLayer.CanCacheTiles = true;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Clear a cached tile images from application memory
+
+The `DeleteTilesFromCache` method used to clear the cached tile images from  application cache memory.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<maps:SfMaps>
+    <maps:SfMaps.Layer>
+        <maps:MapTileLayer x:Name="tileLayer" UrlTemplate="https://tile.openstreetmap.org/{z}/{x}/{y}.png" />
+    </maps:SfMaps.Layer>
+</maps:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+tileLayer.DeleteTilesFromCache();
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Markers
+
+You can add markers in the tile layer. The procedure is very similar to the shape layer. Kindly refer the [markers](https://help.syncfusion.com/maui/maps/markers) section.
```

Comment:
You can add markers in the tile layer. The procedure is very similar to the shape layer. Kindly refer to the [markers](https://help.syncfusion.com/maui/maps/markers) section.

Reply:
Modified the content